server/chat-commands/info.ts

```diff
@@ -2123,20 +2123,15 @@ export const commands: Chat.ChatCommands = {
 	],
 
 	faq(target, room, user) {
-		if (!this.runBroadcast()) return;
 		target = toID(target);
 		const showAll = target === 'all';
-		if (showAll && this.broadcasting) {
-			return this.sendReplyBox(this.tr`You cannot broadcast all FAQs at once.`);
-		}
-
 		const buffer = [];
 		if (showAll || target === 'staff') {
 			buffer.push(`<a href="https://pokemonshowdown.com/${this.tr`pages/staff`}">${this.tr`Staff FAQ`}</a>`);
 		}
 		if (showAll || target === 'autoconfirmed' || target === 'ac') {
 			buffer.push(this.tr`A user is autoconfirmed when they have won at least one rated battle and have been registered for one week or longer. In order to prevent spamming and trolling, most chatrooms only allow autoconfirmed users to chat. If you are not autoconfirmed, you can politely PM a staff member (staff have %, @, or # in front of their username) in the room you would like to chat and ask them to disable modchat. However, staff are not obligated to disable modchat.`);
-			if (!this.broadcasting) void this.parse(`/regtime`);
+			if (!(this.message.startsWith('!') && showAll)) void this.parse(`/regtime`);
```

Comment:
```suggestion
			if (!this.shouldBroadcast()) void this.parse(`/regtime`);
```

Don't use `&& showAll`; instead, move the `showAll` check above this one.